source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,179 @@
+==============
+Retrieve Data
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+database. To retrieve data, use read operations.
+
+Read operations allow you to do the following:
+
+- :ref:`Retrieve a subset of documents from your collection <retrieve-find>`
+- :ref:`Perform transformations on retrieved documents from your collection <retrieve-aggregate>`
+- :ref:`Monitor real-time changes to your database <retrieve-watch>`
+
+Paint Orders
+~~~~~~~~~~~~~
+
+The examples in the following sections are about a paint store that
+keeps track of orders they need to fulfill. The paint store keeps track
+of orders with the ``color`` and ``qty`` fields in their
+``paint_orders`` collection: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "purple", "qty": 10 }
+    { "_id": 2, "color": "green", "qty": 8 }
+    { "_id": 3, "color": "purple", "qty": 4 }
+    { "_id": 4, "color": "green", "qty": 11 }
```

Comment:
I thought it could be assumed that using order in a plural form meant having multiple orders opposed to an array of orders in a document.  I'll change it to singular to avoid confusion/further explanation to the reader.